src/reducers/videoReducer.jsx

```diff
@@ -0,0 +1,11 @@
+export function videoReducer(videoData, action) {
+  console.log(action);
+  switch (action.type) {
+    case "ADD_TO_WATCHLIST":
+      return { ...videoData, watchList: action.payload };
+    case "REMOVE_FROM_WATCHLIST":
```

Comment:
Instead of hardcoded strings, you can set them in a constant variable and use them here.